hazelcast/token_provider.py

```diff
@@ -0,0 +1,38 @@
+
+from six import string_types
```

Comment:
We should use `six` from `hazelcast` package

Reply:
OK: https://github.com/hazelcast/hazelcast-python-client/pull/446/commits/36946e76b40c9e937d31840499db6bd23f1b14c6